### ISSUE TITLE
Can't edit a table

### ISSUE THREAD

**Issue by jlagetz** [sqlitebrowser/sqlitebrowser#2850]
Details for the issue
--------------------

#### What did you see instead? No changes

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [x ] macOS: ( _x version:_11.6___ )
- [ ] Other: ___

#### What is your DB4S version?

- [ ] 3.12.2
- [ ] 3.12.1
- [ ] 3.12.0
- [ ] 3.11.x
- [x ] Other: _Nightly build from 9/12/21

#### Did you also

- [ x] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [x ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

Thank you!

**Comment by justinclift** [comment]
Would it be feasible capture a video of it not working properly?  That should head off most questions, and let us figure out what's going wrong. :smile:

**Comment by jlagetz** [comment]
I've asked the student to send. But I've recorded what she was doing. Mine is successful, hers is not. I thought maybe she had her data in some weird folder without write privileges, but she doesn't.

https://user-images.githubusercontent.com/3595921/135637513-e681a959-d984-4d70-8edc-7bb339b18938.mp4

**Comment by chrisjlocke** [comment]
That video shows a successful edit ... the edited field name is in the new structure?

**Comment by jlagetz** [comment]
Yes - waiting for student video of unsuccessful. But she did exact same tasks without success.

**Comment by chrisjlocke** [comment]
Not the answer, but noticed something 'ooh' while watching the video - you clicked 'OK' before submitting the name change.  This actually worked in this example, but when creating a table (I think) if you click OK before pressing Return (or Enter) on a field, it may not take the name change.  I know if you click 'Add field' without confirming, it doesn't like it.

https://user-images.githubusercontent.com/7872218/135662490-f9d4049e-afe8-4e61-8e54-1a9d8f69c74e.mp4

Apologies - ignore the audio - while screengrabbing, I was listening to the radio, so it recorded that!
